subsys/bluetooth/controller/ll_sw/nordic/lll/lll.c

```diff
@@ -212,6 +212,28 @@ int lll_init(void)
 	return 0;
 }
 
+int lll_deinit(void)
+{
+	int err;
+
+	/* Release clocks */
+	err = lll_clock_deinit();
+	if (err < 0) {
+		return err;
+	}
+
+	/* Disable IRQs */
+	irq_disable(RADIO_IRQn);
+	irq_disable(RTC0_IRQn);
+	irq_disable(HAL_SWI_RADIO_IRQ);
+#if defined(CONFIG_BT_CTLR_LOW_LAT) || \
+	(CONFIG_BT_CTLR_ULL_HIGH_PRIO != CONFIG_BT_CTLR_ULL_LOW_PRIO)
+	irq_disable(HAL_SWI_JOB_IRQ);
+#endif
```

Comment:
Isn't `HAL_SWI_JOB_IRQ` always defined? If so, then we can replace the `#if defined` with a `IS_ENABLED`

Reply:
Yep, fine with that. @cvinayak you OK with this?

Reply:
yes.